docs/pages/management/operations/tls-routing.mdx

```diff
@@ -13,7 +13,7 @@ description: How to upgrade an existing Teleport cluster to single-port TLS rout
   TLS routing is available starting from Teleport `8.0`.
 </Details>
 
-<Admonition scope={["cloud"]} type="tip" scopeOnly>
+<Notice type="tip" label="Teleport Cloud Users">
```

Comment:
Notice components don't include a label, so maybe an Admonition is the way to go here.

Also, with the version warning and two `Notices`, it seems like quite a lot of preparatory messaging before we get to the intro paragraph. Is there another way to organize this?

Reply:
I'm adding the CTA from our CTA partial to the existing notice (also removing the label), and moving it lower to separate from the version warning. FYI This page needs a Refresh anyway (discussed in Slack) since Upgrading to Teleport 8 is no longer news but TLS routing still is, so perhaps another issue should be spawned.